forc-plugins/forc-client/src/op/run/mod.rs

```diff
@@ -116,31 +144,28 @@ pub async fn run_pkg(
         })
         .collect::<Result<Vec<ContractId>>>()?;
 
-    let mut tb = TransactionBuilder::script(compiled.bytecode.bytes.clone(), script_data);
-    tb.maturity(command.maturity.maturity.into())
-        .add_contracts(contract_ids);
-
-    let provider = Provider::connect(node_url.clone()).await?;
-
-    let script_gas_limit = if compiled.bytecode.bytes.is_empty() {
-        0
-    } else if let Some(script_gas_limit) = command.gas.script_gas_limit {
-        script_gas_limit
-    // Dry run tx and get `gas_used`
-    } else {
-        get_script_gas_used(tb.clone().finalize_without_signature_inner(), &provider).await?
```

Comment:
Why is this no longer needed?

Reply:
Because we no longer need to maintain our own signing logic for `WalletUnlocked` as the signer is abstracted as `ForcClientAccount` which implements a `Signer` and `Account` from SDK. The refactor that moves the Wallet into `ForcClientAccount` makes the custom signing logic obsolete. As a byproduct of that, we no longer need to maintain this custom signing logic for scripts. We can simply use SDK to do the signing as `ForcClientAccount` is a valid signer/account from the perspective of the SDK. 

So the refactor we did makes it possible to remove all custom signing logic we have, by moving to use SDK for signatures completly.